sampler_remote.go

```diff
@@ -142,12 +143,17 @@ func (s *RemotelyControlledSampler) Equal(other Sampler) bool {
 }
 
 func (s *RemotelyControlledSampler) pollController() {
-	ticker := time.NewTicker(s.samplingRefreshInterval)
+	expBackOff := backoff.NewExponentialBackOff()
```

Comment:
I believe the way this is written we will exponentially backoff even if we are successfully retrieving the polling file which is not desired. I think we should go with the suggestion in the original issue:

- jitter added to normal polling (to prevent a large number of clients from syncing their requests)
- exponential backoff in the face of errors

Reply:
I think you are right, it will be exponentially even success.  I will reconsider how to archive the original goal.